distribution/bwc/build.gradle

```diff
@@ -161,6 +162,13 @@ bwcVersions.forPreviousUnreleased { BwcVersions.UnreleasedVersionInfo unreleased
         spoolOutput = true
         workingDir = checkoutDir
         doFirst {
+          // TODO: Remove this once we've updated to Gradle 6.1
+          // Workaround for https://github.com/gradle/gradle/issues/11426
+          if (GradleVersion.version(file("${ checkoutDir}/buildSrc/src/main/resources/minimumGradleVersion").text) != GradleVersion.current()) {
+            environment = environment.findAll { key, val -> key != 'OPENSHIFT_IP' }
+            environment.forEach { key, val -> println("$key=$val") }
```

Comment:
Is this leftover debug?

Reply:
Yes, I'll remove this once everything is green.